config/operator/rbac/role.yaml

```diff
@@ -4,6 +4,15 @@ kind: ClusterRole
 metadata:
   name: cl-operator-manager-role
 rules:
+- apiGroups:
```

Comment:
nit: could this be made a namespaced role solely for the CoreDNS config map and the k8s-system namespace? can it be reduced further to only the specific config map?
This can be done by adding [`resourceNames`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources)

Reply:
It doesn't work for me when moving configmap to a namespaced role. I committed the changes under `corednsrewrite-role2` branch derived from this PR branch.

On `AddImport`, ` cl-controlplane` fails with:

```
User "system:serviceaccount:clusterlink-system:cl-controlplane" cannot list resource "configmaps" in API group "" at the cluster scope
E0621 17:56:05.920935       1 reflector.go:150] pkg/mod/k8s.io/client-go@v0.30.2/tools/cache/reflector.go:232: Failed to watch *v1.ConfigMap: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:clusterlink-system:cl-controlplane" cannot list resource "configmaps" in API group "" at the cluster scope
```

Reply:
That's fine for now, then. Please open an issue to narrow permissions from cluster wide to specific namespace/object later and we'll do it in a follow up PR

Reply:
An issue had been opened: https://github.com/clusterlink-net/clusterlink/issues/660